stonesoup/plotter.py

```diff
@@ -40,6 +42,8 @@ class Plotter:
     ----------
     dimension: enum \'Dimension\'
         Optional parameter to specify 2D or 3D plotting. Default is 2D plotting.
+    figsize: tuple
+        Optional parameter to specify the size of the figure. Default is (10, 6).
```

Comment:
I think this should be documented like the other methods.

https://github.com/dstl/Stone-Soup/blob/bfecfd6596ab1d877dc3d895ad3fbce1f7aa459f/stonesoup/plotter.py#L95-L96